catalogue/webapp/pages/api/works/items/[workId].tsx

```diff
@@ -80,10 +80,6 @@ const ItemsApi = async (
   });
   res.setHeader('Content-Type', 'application/json');
   res.setHeader('Access-Control-Allow-Origin', '*');
-  res.setHeader(
```

Comment:
This one should go back in as it's about item availability and so needs to be up-to-date

Reply:
Probably add a comment to this effect as well!